commit abf2030556dfe694dc19ed2d73f84f7e5046b660
Author: Ian Kent <raven@themaw.net>
Date:   Thu Jul 11 13:56:15 2024 +0800

    autofs-5.1.9 - fix amd cache options not copied
    
    The cache options set when parsing the amd map entry are not copied to
    the list entry that gets processed by the caller.
    
    Signed-off-by: Ian Kent <raven@themaw.net>

---
 CHANGELOG           |    1 +
 modules/amd_parse.y |    1 +
 2 files changed, 2 insertions(+)

--- autofs-5.1.4.orig/CHANGELOG
+++ autofs-5.1.4/CHANGELOG
@@ -167,6 +167,7 @@
 - fix always recreate credential cache.
 - fix missing unlock in sasl_do_kinit_ext_cc().
 - handle sss special case getautomntbyname() error.
+- fix amd cache options not copied.
 
 xx/xx/2018 autofs-5.1.5
 - fix flag file permission.
--- autofs-5.1.4.orig/modules/amd_parse.y
+++ autofs-5.1.4/modules/amd_parse.y
@@ -888,6 +888,7 @@ static int add_location(void)
 		new->path = entry.path;
 	}
 	new->flags = entry.flags;
+	new->cache_opts = entry.cache_opts;
 	new->type = entry.type;
 	new->map_type = entry.map_type;
 	new->pref = entry.pref;
